core/src/main/java/org/elasticsearch/common/geo/GeoUtils.java

```diff
@@ -56,6 +65,30 @@
     /** Earth ellipsoid polar distance in meters */
     public static final double EARTH_POLAR_DISTANCE = Math.PI * EARTH_SEMI_MINOR_AXIS;
 
+    /** Returns true if value is neither NaN nor infinite. */
+    public static boolean isValidNumber(double value) {
```

Comment:
Couldn't make it package private - as it turns out I also needed it for validation in the GeoBoundingBoxQueryBuilder. Moved it to oe.common.Numbers - at least should make more sense there then in GeoUtils I think.